src/styles/utilities/display.ts

```diff
@@ -28,4 +28,4 @@ export default {
     dBlock: {
         display: 'block',
     },
-};
+} as const
```

Comment:
I could find a way to make `satisfies Record<string, ViewStyle>` work here, as some of these **display** values are not supported by the type

Reply:
Looks like it's because of the RNWeb styles that are not present in RN. Would it work if we extended `ViewStyle` with the remaining values?

```ts
satisfies Record<string, ViewStyle & {display: 'inline' | 'inline-flex' | 'block'}>
```

Reply:
nice ⬆️

Reply:
Done!